verify/verify.go

```diff
@@ -1301,6 +1331,43 @@ func TdxQuote(quote any, options *Options) error {
 	}
 }
 
+// SupportedTcbLevelsFromCollateral returns the matching TCB levels of TDX TCB and QE Identity respectively by checking collaterals.
+// It is the caller's responsibility to verify the quote before calling this function.
+func SupportedTcbLevelsFromCollateral(quote any, options *Options) (tdxtcb pcs.TcbLevel, qe pcs.TcbLevel, err error) {
+	if options == nil {
+		err = ErrOptionsNil
```

Comment:
for better readability can we return like `return pcs.TcbLevel{}, pcs.TcbLevel{}, ErrOptionsNil`, below as well

Reply:
done